internal/cloud/realm/dependencies.go

```diff
@@ -102,7 +102,7 @@ func (c *client) ImportDependencies(groupID, appID, uploadPath string) error {
 }
 
 func (c *client) ExportDependencies(groupID, appID string) (string, io.ReadCloser, error) {
-	res, resErr := c.do(http.MethodGet, fmt.Sprintf(dependenciesArchivePathPattern, groupID, appID), api.RequestOptions{})
+	res, resErr := c.do(http.MethodGet, fmt.Sprintf(dependenciesExportPathPattern, groupID, appID), api.RequestOptions{})
```

Comment:
[opt] non-named error here

Reply:
per previous comment will just keep it, lemme me know if this is not OK.